src/schema/json_object_options.rs

```diff
@@ -13,7 +13,8 @@ pub struct JsonObjectOptions {
     // If set to some, int, date, f64 and text will be indexed.
     // Text will use the TextFieldIndexing setting for indexing.
     indexing: Option<TextFieldIndexing>,
-
+    // Store all field as fast fields.
+    fast: bool,
```

Comment:
I think a list of paths for which to create fast fields would be good

Reply:
Not in tantivy... but I think at one point we might want some router indeed.